babel/core.py

```diff
@@ -60,6 +60,9 @@ def get_global(key):
             fileobj.close()
     return _global_data.get(key, {})
 
+# FIXME: Very ugly hack in order to break cyclic import dependency
+localedata.get_global = get_global
```

Comment:
Could we add the import statement inside `babel.localedate.load()` instead?

PS: I didn't tested it, just an idea.